Update access instructions for Azure VPN client setup (#11)
Uniphar/devops-guidelines: AzureVpnClientSetup.md

# Configure the Azure VPN client

## Pre-requisites

> Make sure you have been added to the **uni-sec-VPN_Users** group in Azure AD.

Reviewer: ```suggestion
Make sure you have been added to the [uni-sec-VPN_Users](https://portal.azure.com/#view/Microsoft_AAD_IAM/GroupDetailsMenuBlade/~/Overview/groupId/82ab6a21-ece0-4ac0-b411-67774c886ead/menuId/) group in Azure AD.
```

profile configuration file. This file contains all the information needed to
connect to the VPN gateway.

> These profile configuration files are XML files can be found by navigating to

Reviewer: ```suggestion
These profile configuration files are XML files that can be found by navigating to
```

`\\unicoreplatform.file.core.windows.net\vpnxmls` in file explorer. If you have
correctly been added to the Azure VPN Users Entra ID Group, you should be able

> to access without being prompted to authenticate. If you are prompted to authenticate,

Reviewer 2: ```suggestion
to access the files without being prompted to authenticate. If you are prompted to authenticate,
```

> to access without being prompted to authenticate. If you are prompted to authenticate,
> please contact the TechOps team and confirm weather you are in the group.

Reviewer: ```suggestion
to access without being prompted to authenticate. You may need to wait for
your group membership to propagate, but if after an hour you still cannot
access the share, please contact the TechOps team and confirm whether
you are in the group.
```
